src/mono/wasm/runtime-test.js

```diff
@@ -333,24 +344,23 @@ async function loadDotnet(file) {
             const script = document.createElement("script");
             script.src = file;
             document.head.appendChild(script);
-            return globalThis.Module;
+            while (true) {
+                if (globalThis.createDotnetRuntime) {
```

Comment:
This infinite loop needs a bailout condition (a timeout is probably fine)

Reply:
This whole `loadDotnet` loader will be replaced by single `import` in ES6 for all engines. Added timeout for now.